src/applications/personalization/view-dependents/reducers/index.js

```diff
@@ -0,0 +1,37 @@
+import { splitPersons } from '../util';
+
+import {
+  FETCH_ALL_DEPENDENTS_SUCCESS,
+  FETCH_ALL_DEPENDENTS_FAILED,
+} from '../actions';
+
+const initialState = {
+  loading: true, // app starts in loading state
+  error: null,
+  onAwardDependents: null,
+  notOnAwardDependents: null,
+};
+
+let allPeople = null;
```

Comment:
Would it be better to just keep this list in state, or in the reducer? Sort of feels like it shouldn't be global.